### DISCUSSION TITLE
Why does it not find my roms, even tho it's allowed in the location where the rom is?

### DISCUSSION THREAD

**Discussion by j0j0p0se** [skyline-emu/skyline#1060] [Q&A] [answered]
I have that problem where it can't find my rom which really annoys me since I made 2 copies of one rom in the location which made my storage full. Why does it do that tho?

**Comment by yhak22** [comment, marked as answer]
check if you dump your game correctly it should be in .xci format at least or other format that support the app 
check the format types in the app and make sure your game is one of them.
